packages/opentelemetry-api/src/trace/span.ts

```diff
@@ -25,18 +25,26 @@ import { TimeInput } from '../common/Time';
  * in-process function calls to sub-components. A Trace has a single, top-level
  * "root" Span that in turn may have zero or more child Spans, which in turn
  * may have children.
+ *
+ * Spans are created by the {@link Tracer.startSpan} method.
  */
 export interface Span {
   /**
    * Returns the {@link SpanContext} object associated with this Span.
    *
+   * Get an immutable, serializable identifier for this span that can be used
```

Comment:
Is the spancontext object actually immutable? I know it is a copy, so changing its fields doesn't affect the span. But I don't think it is actually immutable.

Reply:
Think in the end this is a quibble which doesn't matter. Going to mark as resolved.

Reply:
As per the [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#spancontext): `SpanContext`s are immutable. `SpanContext` MUST be a final (sealed) class.

Reply:
Right, but I think our implementation is just an Object which can be modified. In the `Span` implementation in `tracing`, the span context is just a private property on the span.

I think it's not a problem with this PR, but we should probably have a `SpanContext` class in `tracing` which enforces immutability.